### ISSUE TITLE
[Android]Can't tap with Frame in CollectionView on Android

### ISSUE THREAD

**Issue by thaihuydang** [dotnet/maui#7121]
### Description

When I add Frame control in data template in collection view, I can't tap on collection view.

### Steps to Reproduce

1. Create Maui App
2. Add collection view
3. add frame to colleciton view

### Version with bug

Release Candidate 3 (current)

### Last version that worked well

Unknown/Other

### Affected platforms

Android

### Affected platform versions

Android 11

**Comment by VincentBu** [comment]
not repro with vs main build(32512.449.main). Here is an example. @thaihuydang , would you like to verify it with your vs?
[MauiApp7121.zip](https://github.com/dotnet/maui/files/8684966/MauiApp7121.zip)

**Comment by RedChops** [comment]
FWIW, it doesn't work for me either. I switched to using Border in my app instead of Frame so my testing is limited. The SelectedItem command is definitely not called, I don't think the tapped event is fired either.

Some events work though, if I set CollectionView.CanReorderItems = True, I can reorder the frames, but still not tap them and get the event. 

The sample app from @VincentBu uses a gesture recognizer within an element inside the frame inside the CollectionView - those gesture recognizers do work for me, what doesn't work is the selection events on the CollectionView itself when using Frame

**Comment by thaihuydang** [comment]
> FWIW, it doesn't work for me either. I switched to using Border in my app instead of Frame so my testing is limited. The SelectedItem command is definitely not called, I don't think the tapped event is fired either.
> 
> Some events work though, if I set CollectionView.CanReorderItems = True, I can reorder the frames, but still not tap them and get the event.
> 
> The sample app from @VincentBu uses a gesture recognizer within an element inside the frame inside the CollectionView - those gesture recognizers do work for me, what doesn't work is the selection events on the CollectionView itself when using Frame

Yes, exactly with my vs

**Comment by IeuanWalker** [comment]
Ye same issue here - 
```xaml
<CollectionView FlowDirection="RightToLeft"
                ItemSizingStrategy="MeasureFirstItem"
                ItemsLayout="HorizontalList"
                ItemsSource="{Binding Dates}"
                SelectedItem="{Binding SelectedDiary}"
                SelectionMode="Single">
    <CollectionView.ItemTemplate>
        <DataTemplate>
            <ContentView Padding="10" x:DataType="viewModel:DiaryTestModel">
                <Frame Padding="0"
                        BackgroundColor="#E5F7FF"
                        CornerRadius="20"
                        HasShadow="False"
                        IsEnabled="false">
                    <VerticalStackLayout Padding="10">
                        <Label HorizontalTextAlignment="Center"
                                Text="{Binding DayOfTheWeekLetter}"
                                TextColor="#00A2EE" />
                        <Label HorizontalTextAlignment="Center"
                                Text="{Binding Date, StringFormat='{0:dd}'}"
                                TextColor="Black" />
                    </VerticalStackLayout>
                </Frame>
            </ContentView>
        </DataTemplate>
    </CollectionView.ItemTemplate>
</CollectionView>
```

**Comment by VincentBu** [comment]
Yes, Item selection event doesn't triggered. `SelectionChanged` method isn't called.

**Comment by xrail91** [comment]
Only the area with the Frame is not triggering the Event. My Frame has a little margin on it and if you click outside the frame the event / command fires.

Hope this is helpfull.

![collectionView_Frame](https://user-images.githubusercontent.com/60090746/168912681-6df9c2b5-3f64-423c-b8af-799ab313782f.gif)

**Comment by ederbond** [comment]
Workaround: Replace the frame with a <Border>. 
The <Border> element it's works fine.

**Comment by Redth** [comment]
As mentioned, there's a workaround which is to use a Border.  We generally recommend this as opposed to the frame anyway.  You can achieve the same rounded corners and/or shadow effect with a Border.

**Comment by mattleibow** [comment]
This method intercepts the touch:

https://github.com/dotnet/maui/blob/main/src/Controls/src/Core/Compatibility/Handlers/Android/FrameRenderer.cs#L171-L179

Then is blocked here:

https://github.com/dotnet/maui/blob/main/src/Controls/src/Core/Compatibility/Android/Renderers/MotionEventHelper.cs#L18-L26

This method returns true. 

In Xamarin.Forms, this line is extra:
https://github.com/xamarin/Xamarin.Forms/blob/5.0.0/Xamarin.Forms.Platform.Android/Renderers/MotionEventHelper.cs#L24


That might be the issue?

**Comment by suchithm** [comment]
Any update? Almost one year (Jun/2023) same issue, seeing ToDo tag?

**Comment by jfversluis** [comment]
I think this might be fixed by #14846 @mattleibow

**Comment by hartez** [comment]
Just tried the repro project on 17.6.4, and the OnTapped method is called when I tap an item in the CollectionView. Not sure which specific PR fixed it, but as far as I can tell this works fine on 7.0.
